tests/countries/test_india.py

```
@@ -10,566 +10,1147 @@
+            name,
+            "2020-08-01",
+            "2021-07-21",
+            "2022-07-10",
+            "2023-06-29",
+            "2024-06-17",
             "2025-06-07",
         )
-        self.assertHolidayName(name, dt)
+        self.assertIslamicNoEstimatedHolidayName(name, self.full_range)
+
+    def _assertHinduHolidayHelper(  # noqa: N802
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_assertHinduHolidayHelper`

Add return type annotation: `None`

(ANN202)